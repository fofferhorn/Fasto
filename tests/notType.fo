fun bool main() =
    write(not 3)