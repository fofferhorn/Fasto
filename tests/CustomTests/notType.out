Type error: Cannot unify types bool and int at line 2 column 11